compatibility/common/src/main/java/org/projectnessie/tools/compatibility/internal/CurrentNessieServer.java

```diff
@@ -61,7 +61,7 @@ static CurrentNessieServer currentNessieServer(
 
   @Override
   public URI getUri(Class<? extends NessieApi> apiType) {
-    return Util.resolveNessieUri(jersey.getUri().resolve("api/"), apiType);
+    return Util.resolveNessieUri(jersey.getUri(), serverKey.getVersion(), apiType);
```

Comment:
https://github.com/projectnessie/nessie/pull/7854 added `api/` here but nothing version-aware in `OldNessieServer`

any ideas how to stay "ahead" of such problems in the future?
i.e. do we need server/client compatibility tests that use a `Version.LAST_RELEASE` value? then at least after a new release we might get notified.

or a test that uses `Version.CURRENT` with `OldNessieServer` somehow?

Reply:
`OldNessieServer` relies on published jars, so it is currently at least one release behind (it cannot test `main` due to lack of artifacts). One option may be to do `publishToMavenLocal` and use those artifacts in CI.

Reply:
Add the version to `nessie-compatibility.properties`?

Reply:
If we go with `publishToMavenLocal` I believe so extra changes will be required to support "snapshot" versions.

Reply:
> Add the version to nessie-compatibility.properties?

doesnt this require awareness of the developer that made the change that compatibility was broken?
CI tests should find this for the developer "automatically"